src/lib/y2partitioner/widgets/lvm_vg_devices_selector.rb

```diff
@@ -38,6 +38,26 @@ def initialize(controller)
         super()
       end
 
+      def help
+        help_available_pvs + help_selected_pvs
+      end
+
+      def help_available_pvs
+        _("<p><b>Available Devices:</b> " \
+          "A list of available LVM physical volumes. " \
```

Comment:
Strictly speaking, not only partitions can be used. In general, not used devices (disks, raids, etc) can be selected as PV. Maybe this description should be similar to the the description in "md_devices_selector". See https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2partitioner/actions/controllers/lvm_vg.rb#L147

Reply:
Strictly speaking, it does not say here that this is limited to partitions. It just shows one way to add devices if there are none to choose from, and since it's pretty hard to insert additional physical hard disks at this stage, the next paragraph suggest the one thing that is doable with a reasonable amount of work for the user: Create dedicated LVM PV partitions.

This help is not meant to be a complete substitute for the manual; it is supposed to give some hints. Unlike the old help texts that were pretty much useless and explained only the obvious things (which is of course a lot easier to write), my intention was to point the user into the right direction what can be done. Yes, this might mean that it does not cover every single corner case; that's life. But an understandable text is much better than a lawyer-like complete specification of all the possible scenarios and pathological cases; clarity wins over completeness.

Reply:
Completely agree, but I was not suggesting to add a fully complete description of all possible cases, only some description in the way you did for md devices. But I agree, at least it is told the easy way to have available PVs.